website/docs/apps/launcher/README.md

```diff
@@ -80,7 +80,39 @@ It is designed to capture and store raw data from various robot
 [**topics**](/docs/concepts/recorder_service/#available-topics-for-recording),
 facilitating later analysis, debugging, and model training.
 
-![recorder](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/42edc61f-b7ef-4a7c-9831-90523a1d83ee)
+![recorder](https://github.com/user-attachments/assets/68cabb56-1b16-4c45-a047-202910879873)
+
+With OS 2.3.1 comes Recorder Profiles.
+
+Users can now create multiple profiles to record from, allowing the flexibility to swap which topics get recorded,
+without having to overwrite previously saved configurations.
+
+To get started, click **"New Profile"**, enter a name, and click **"Save Profile"**. This is now the currently
+selected profile. <br/>
+
+![recorder new profile](https://github.com/user-attachments/assets/bc24d92d-bc85-4072-9e96-fe5667ad7792)
+
+*(As of this release you must create a profile in order to save topics. Attempting to save topics without a
+current profile will prompt the profile creation flow)*
+
+To edit a profile, simply make changes to it while it is the current profile, and click **"Save Topics"**.
```

Comment:
The text on the right-hand side of the profile name reads "Active Profile". So I think it's best if we consistently use "Active Profile" instead of "Current Profile"